geometry_msgs/PoseStamped[] poses
geometry_msgs/PointStamped[] landmarks
